lib/iex/test/iex/config_test.exs

```diff
@@ -0,0 +1,9 @@
+Code.require_file "../test_helper.exs", __DIR__
+
+defmodule IEx.ConfigTest do
+  use ExUnit.Case, async: true
+
+  test "configuration defaults width to 80" do
```

Comment:
It'd be better to remove `80` from the test title, thus if default changes we don't need to rename it as well.